lib/datadog/core/environment/variable_helpers.rb

```diff
@@ -7,54 +7,36 @@ module Environment
       # Defines helper methods for environment
       # @public_api
       module VariableHelpers
```

Comment:
Note that APIs on this file are marked as "@public_api" which means that customers are allowed to use them, and we're not supposed to break them between minor releases.

This doesn't mean you can't apply your changes. One possibility is to make sure that the old versions of the methods are at least still available (and probably marked as deprecated).